csrc/scheduler/mma_utils.cpp

```diff
@@ -1229,6 +1229,86 @@ RolesMapOpt getTensorsRoles(Fusion* fusion) {
   return roles_map;
 }
 
+namespace {
+
+void add_mma_op_helper(TensorView* tv_a, TensorView* tv_b, TensorView* out) {
+  auto* init = IrBuilder::create<Val>(0.0);
+
+  NVF_CHECK(
+      tv_a->getDataType().value() == DataType::Half ||
+      tv_a->getDataType().value() == DataType::BFloat16);
```

Comment:
I think it would be better to let the `MmaOp` handle checks like this. Also we'll want single precision, and potentially others in the future.

Reply:
Thanks, removed.